state/execution.go

```diff
@@ -487,12 +487,14 @@ func buildExtendedCommitInfo(ec *types.ExtendedCommit, store Store, initialHeigh
 				panic(fmt.Errorf("commit at height %d received with missing vote extensions data", ec.Height))
 			}
 			ext = ecs.Extension
+			extSig = ecs.ExtensionSignature
```

Comment:
in line 434, above, Tendermint -> CometBFT

Reply:
Thanks for spotting. I'm changing this in an upcoming related PR, which I am doing separately to avoid bloating this one.

Reply:
#509